<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>530</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textDisplay">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>481</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputField">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>481</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipEdit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>90</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendEdit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>150</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="rcvEdit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>290</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectFileButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>selectFile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>0</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
